packages/salesforcedx-vscode-core/src/commands/commands.ts

```diff
@@ -120,6 +121,49 @@ export type DirFileNameSelection = {
   outputdir: string;
 };
 
+export type TempFile = {
+  fileName: string;
+};
+
+export class CreateApexTempFile
```

Comment:
This could probably be moved into the forceApexExecute.ts file.
I don't think there is much reuse for this outside for execute anonymous.
If we do find that we have reuse we can move it back here.

Reply:
I'll move the whole TempFile/CreateApexTempFile to the forceApexExecute.ts file. Was just following the existing pattern. ;-)